### PR TITLE
fix(settings): re-add back to project button

### PR DESCRIPTION
**Pull request by Chrissy**
image

### CODE REVIEW

File: src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import {Link} from 'react-router';
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx]: this file is a dup of the old settingsBackButton component, minus the marked style block;

File: src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import styled from 'react-emotion';
+
+import {t, tct} from 'app/locale';
+import InlineSvg from 'app/components/inlineSvg';
+import SentryTypes from 'app/sentryTypes';
+import replaceRouterParams from 'app/utils/replaceRouterParams';
+import withLatestContext from 'app/utils/withLatestContext';
+import space from 'app/styles/space';
+
+const BackButtonWrapper = styled(Link)`
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx]: this style block has been retooled a bit

File: src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  background: ${p => p.theme.whiteDark};
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+  padding: ${space(1.5)} ${space(2)};
+  font-size: ${p => p.theme.fontSizeMedium};
+  &:hover {
+    color: ${p => p.theme.gray5};
+  }
+`;
+
+const Icon = styled(InlineSvg)`
```
Comment (Chrissy) [inline src/sentry/static/sentry/app/views/settings/components/settingsBackButton.jsx]: same with this one

**Comment by EvanPurkhiser** [comment]
Oooh, this looks really nice

**Review by ckj — APPROVED** [review]
Nicely done

**Pull request merged**